src/modals/newsletter/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const IMAGE_HEIGHT = 180;
```

Comment:
use formatSize()

Reply:
We need exactly 180px height because of svg image sizes. We have the same logic on `notification-item` component page